front/app/components/FormBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -123,22 +123,25 @@ export const ContentSettings = ({
             buttonStyle="secondary"
             onClick={onClose}
             minWidth="160px"
+            width={field.isDeleteEnabled === false ? '100%' : 'auto'}
```

Comment:
I think the issue is that "isDeleteEnabled" isn't always true/false :thinking: If it's undefined (since it's optional), `get(field, 'isDeleteEnabled', true)` would return "false" right? Which then causes isDeleteEnabled to incorrectly equal "false"?

Reply:
Or maybe I've misunderstood?? That seemed to be the issue when I tried implementing it this way though:

`width={!field.isDeleteEnabled ? '100%' : 'auto'}`

Reply:
The third parameter true is the default value so it would be what is set if something is set and true if nothing is set. You can also see https://lodash.com/docs/4.17.15#get